stack/vagrant/Vagrantfile

```diff
@@ -76,7 +76,7 @@ Vagrant.configure("2") do |config|
       v.memory = 2048
       v.cpus = 2
       v.gui = true
-      v.customize ["modifyvm", :id, "--nic1", "hostonlynet", "--nic2", "nat", "--boot1", "disk", "--boot2", "net"]
```

Comment:
For me, this change causes the VM not to start and give the error:

```bash
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "49c31a7a-eb66-46cb-a5ba-b75befce8dd8", "--type", "gui"]

Stderr: VBoxManage: error: Host-only adapters are no longer supported!
VBoxManage: error: For your convenience a host-only network named 'Legacy  Network' has been created with network mask '255.255.255.0' and IP address range '192.168.56.1' - '192.168.56.254'.
VBoxManage: error: To fix this problem, switch to 'Host-only Network' attachment type in the VM settings.
VBoxManage: error:  (VERR_NOT_FOUND)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

Reply:
I think I messed it up and accidentally closed the pull request. The `hostonlynet` portion works only on a mac. https://www.virtualbox.org/ticket/21148

Not sure if there is a way to detect if it is on a mac and use `hostonlynet` and non mac to use `hostonly`.